rust/core-lib/Cargo.toml

```diff
@@ -16,6 +16,13 @@ xi-rope = { path = "../rope", version = "0.2" }
 xi-unicode = { path = "../unicode", version = "0.1.0" }
 xi-rpc = { path = "../rpc", version = "0.2.0" }
 
+[dependencies.syntect]
```

Comment:
This will have to be a release, as we can't publish core-lib to crates.io if it has git dependencies.